indico/htdocs/sass/modules/_category.scss

```diff
@@ -418,3 +418,160 @@ ul.category-resource-qtip {
         }
     }
 }
+
+.category-overview {
+    > .title {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: 30px;
+
+        .text {
+            font-size: 1.2em;
+            font-style: italic;
+            margin-left: 5px;
+            margin-right: 5px;
+        }
+
+        .navigation a {
+            padding: 5px;
+        }
+    }
+
+    > table:not(.day) {
+        border: 1px solid #aaa;
+        border-collapse: separate;
+        border-spacing: 1px;
+
+        tr {
+            background-color: #eee;
+        }
+    }
+
+    table {
+        width: 100%;
+        border-collapse: collapse;
+
+        th {
+            background-color: #444;
+            color: #fff;
+            text-align: center;
+            font-weight: normal;
+            white-space: nowrap;
+            padding: 1px 15px;
+        }
+    }
+
+    td {
+        padding: 0;
+        vertical-align: top;
+
+        &.day {
+            vertical-align: top;
+            padding: 5px;
+
+            &:not(:empty) {
+                min-width: 180px;
+            }
+
+            > .calendar-date {
+                font-size: 1.5em;
+                font-weight: bold;
+                text-align: center;
+
+                a {
+                    color: #aaa;
+                }
+            }
+
+            table {
+                tr {
+                    background-color: white;
+
+                    td {
+                        font-size: inherit;
+
+                        &.content-info {
```

Comment:
Yeah, overridability makes more sense for reusable things, not styles specific to one legacy page